apps/openmw/mwmechanics/character.cpp

```diff
@@ -678,76 +651,12 @@ void CharacterController::refreshCurrentAnims(CharacterState idle, CharacterStat
 }
 
 
-void getWeaponGroup(WeaponType weaptype, std::string &group)
+void getWeaponGroup(const int weaptype, std::string &group)
```

Comment:
This is not a class member. Do we care if this function still exists if it's possible to just recover the group directly in the same amount of code?